Wiki/Programs/PokemonSV/TeraAutoHost.md

```diff
@@ -0,0 +1,56 @@
+# Tera Raid AutoHost in Pokemon Scarlet and Violet
+**CAPTURE CARD REQUIRED**
+
+**Related Programs:**
+- **Microcontroller:** [Tera Raid AutoHost](https://github.com/Vaedz7/Microcontroller/blob/master/Wiki/Programs/PokemonSV/TeraAutoHost.md) (this program)
+- **Computer Control:** [Tera Auto-Host](https://github.com/PokemonAutomation/ComputerControl/blob/master/Wiki/Programs/PokemonSV/AutoHost.md)
+
+## Program Description
+
+Automatically Host Tera Raids for Discord/Twitch without the use of a Serial Connection!
+
+## Setup of Settings
+### Board Setup
+1. Follow Microcontroller guide by Pokemon Automation [here](https://github.com/PokemonAutomation/Microcontroller "here")
+2. Download program(s) from **THIS REPOSITORY**
+3. Copy and Paste ``TeraAutoHost_Core.c`` to ``CustomProgram_Core.c``

```

Comment:
that step is missing for other programs ?

Reply:
That isn't supposed to be there. Removed.